README.md

```diff
@@ -1 +1 @@
-# covidiena
+# covidiena
```

Comment:
what changes were made to this? looks like the same

Reply:
I was adding and removing so ensure that it was working for me before I made major changes